test/demo-autoresearch/experiment.py

```
@@ -0,0 +1,68 @@
+    """Predict y given x. Modify this function to minimize MSE."""
+    return 0.0
+
+# ── Immutable: evaluation harness ──────────────────────────
+def evaluate():
+    start = time.time()
+    errors = []
+    for x, y_true in DATA:
+        try:
+            y_pred = predict(x)
+            errors.append((y_pred - y_true) ** 2)
+        except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)